src/main/java/com/uwetrottmann/tmdb/Tmdb.java

```diff
@@ -166,4 +158,5 @@ public DiscoverService discoverService() {
         return getRestAdapter().create(DiscoverService.class);
     }
 
+    public CollectionService collectionService() { return getRestAdapter().create(CollectionService.class); }
```

Comment:
No single line methods, please use new lines (see [Google Java Style](https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.1-braces) for a good starting point).